oml/datasets/texts.py

```diff
@@ -0,0 +1,239 @@
+from copy import deepcopy
+from typing import Any, Dict, List, Optional, Union
+
+import numpy as np
+import pandas as pd
+from torch import BoolTensor, LongTensor
+
+from oml.const import (
+    BLACK,
+    CATEGORIES_COLUMN,
+    INDEX_KEY,
+    INPUT_TENSORS_KEY,
+    IS_GALLERY_COLUMN,
+    IS_QUERY_COLUMN,
+    LABELS_COLUMN,
+    LABELS_KEY,
+    SEQUENCE_COLUMN,
+    TEXTS_COLUMN,
+    TColor,
+)
+from oml.interfaces.datasets import (
+    IBaseDataset,
+    ILabeledDataset,
+    IQueryGalleryDataset,
+    IQueryGalleryLabeledDataset,
+    IVisualizableDataset,
+)
+from oml.utils.misc import visualise_text
+
+THuggingFaceTokenizer = Any
```

Comment:
May be we should not specify tokeriner as HF? Or use specific class from tokenizers (it already installed with hf). from tokenizers import Tokenizer

Reply:
Found that comment below. I see. May be add it here too?

`# We don't use actual types programmatically in order not to bring optional dependency here`

Reply:
okay, updated types here and for HFWrapper

Reply:
I've added runtime type checking